plat/kvm/Makefile.uk

```diff
@@ -71,7 +71,6 @@ LIBKVMPLAT_SRCS-$(CONFIG_ARCH_X86_64) += $(LIBKVMPLAT_BASE)/x86/setup.c
 LIBKVMPLAT_SRCS-$(CONFIG_ARCH_X86_64) += $(LIBKVMPLAT_BASE)/x86/console.c
 LIBKVMPLAT_SRCS-$(CONFIG_ARCH_X86_64) += $(LIBKVMPLAT_BASE)/x86/lcpu.c
 LIBKVMPLAT_SRCS-$(CONFIG_ARCH_X86_64) += $(LIBKVMPLAT_BASE)/x86/lcpu_start.S
-LIBKVMPLAT_SRCS-$(CONFIG_ARCH_X86_64) += $(LIBKVMPLAT_BASE)/x86/intctrl.c
```

Comment:
In line 12 above you can also remove `-$(eval $(call addplatlib_s,kvm,libkvmgic,$(CONFIG_LIBGIC)))`.

Also further down:
```
@@ -197,19 +196,3 @@ LIBKVMVIRTIO9P_ASINCLUDES-y   += -I$(UK_PLAT_DRIVERS_BASE)/include
 LIBKVMVIRTIO9P_CINCLUDES-y    += -I$(UK_PLAT_DRIVERS_BASE)/include
 LIBKVMVIRTIO9P_SRCS-y +=\
                        $(UK_PLAT_DRIVERS_BASE)/virtio/virtio_9p.c
-
-##
-## GIC library definitions
-##
-ifeq ($(findstring y,$(CONFIG_LIBGIC)),y)
-LIBKVMGIC_CINCLUDES-y         += -I$(LIBKVMPLAT_BASE)/include
-LIBKVMGIC_CINCLUDES-y         += -I$(UK_PLAT_COMMON_BASE)/include
-LIBKVMGIC_CINCLUDES-y         += -I$(UK_PLAT_DRIVERS_BASE)/include
-LIBKVMGIC_SRCS-y += $(UK_PLAT_DRIVERS_BASE)/gic/gic-common.c
-ifeq ($(findstring y,$(CONFIG_LIBGICV2)),y)
-LIBKVMGIC_SRCS-y += $(UK_PLAT_DRIVERS_BASE)/gic/gic-v2.c
-endif
-ifeq ($(findstring y,$(CONFIG_LIBGICV3)),y)
-LIBKVMGIC_SRCS-y += $(UK_PLAT_DRIVERS_BASE)/gic/gic-v3.c
-endif
-endif
```